appveyor.yml

```diff
@@ -19,6 +19,7 @@ matrix:
   fast_finish: true
 install:
   - ps: Install-Product node $env:nodejs_version x64
+  - cmd: npm i -g npm@latest
```

Comment:
It's to fix appveyor on node 4, which uses npm 2.x which apparently isn't compatible with babel